### PR TITLE
Add warning about NorthstarProton breakage

### PR DESCRIPTION
**Pull request by Jan200101**
made quickly using the Github web editor

Adds a warning about NorthstarProton fucking itself up.

### CODE REVIEW

**Comment by Jan200101** [comment]
@cyrv6737

File: docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 EA App currently displays a blank screen when using NorthstarProton, however Northstar will still launch assuming you have logged in to EA App at least once.
 {% endhint %}
 
+{% hint style="warning" %}
+NorthstarProton has some problems and may stop working at any point, if this happens you can try deleting the compatibility directory or visit the Northstar Discord for help.
```
Comment (itscynxx) [inline docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md]:
```suggestion
NorthstarProton has some problems and may stop working at any point, if this happens you can try [deleting the compatibility directory](linux-troubleshooting.md#northstar-not-launching-with-steam) or visit the Northstar Discord for help.
```

**Review by itscynxx — COMMENTED** [review]
may as well link to an explanation of it, confirmed this link works on my fork and that gitbook allows for hyperlinks in hint text, as you can see it working on the [server hosting guides](https://r2northstar.gitbook.io/r2northstar-wiki/hosting-a-server-with-northstar/guides) page

**Review by GeckoEidechse — APPROVED** [review]
Looks fine as far as I can tell. Added content is also correct from what I know from discussions in Linux channel on Northstar Discord

**Pull request merged**